pkgs/conf-pkg/changelog/2022/JANUARY.rst

```diff
@@ -0,0 +1,7 @@
+--------------------------------------------------------------------------------
```

Comment:
Please move this file to be in `genielibs/pkgs/conf-pkg/changelog/undistributed/`

Also, please rename it so that it has a unique name in case other changelogs get added in other PRs this month. Something like `genielibs/pkgs/conf-pkg/changelog/undistributed/changelog_fix_iosxe_interface_order_20220114.rst` would be good

Reply:
It seems that we're missing the documentation that explains how to handle changelogs. I'll add it to the README soon